thirdeye-ui/src/app/rest/dto/anomaly.interfaces.ts

```diff
@@ -61,6 +61,7 @@ export interface AnomalyFeedback {
         principal: string;
     };
     updated?: number;
+    updateAlertConfiguration?: "ASYNC";
```

Comment:
nit. you can use enum if you think this will have different modes. If no, you can simply use boolean?

Reply:
As of now, this option can just take one value, in the future it would support multiple modes. Having just one mode right now, I added that as a the only specific type value it can take.